### DISCUSSION TITLE
NOTE - 1. How to en-able TEXT-WRAPPING for NOTEs?

### DISCUSSION THREAD

**Discussion by Damianus500** [freeplane/freeplane#717] [Help]
I am using Freeplane 1.10.3.

Very happy that I could use markdown even in the NOTEs of each node.

2 questions:

1. HOW TO en-able text-wrapping in the NOTE edit box?

2. Possible to edit in an external editor (i.e. pictures could be viewed via an external editor.)

Any advice much appreciated. 

Thank you!

Damianus

**Comment by euu2021** [comment]
>HOW TO en-able text-wrapping in the NOTE edit box?

I think this is possible in the latest versions. See #614 

>Possible to edit in an external editor (i.e. pictures could be viewed via an external editor.)

Take a look at #408

**Comment by dpolivaev** [comment]
Implemented in freeplane 1.10.5 (currently only previews)

**Comment by autoca** [comment]
You can also replace `<body>` with, e.g., ` <body width=80%>` in HTML Code view, or to any other value, to make the text width even less wide than the edit area. Maybe this also helps?

**Reply by autoca** [reply]
BTW. in order to make this usage of `width=80%` in a note a bit easier, the following custom html can be placed in the note of a predefined style (ofc, in html code view):
```
<html>
  <head>
    
  </head>
  <body width="80%">
    <p>
      placeholder
    </p>
  </body>
</html>
```
A placeholder text must be used - otherwise the custom html is lost.
Then a new node in the map must have that style assigned and then chose Edit > Node properties > Copy extensions from style node.
